controllers/construct/mongodbstatefulset.go

```diff
@@ -53,13 +54,14 @@ const (
 	VersionUpgradeHookImageEnv      = "VERSION_UPGRADE_HOOK_IMAGE"
 	ReadinessProbeImageEnv          = "READINESS_PROBE_IMAGE"
 	agentLogLevelEnv                = "AGENT_LOG_LEVEL"
+	agentLogFileEnv                 = "AGENT_LOG_FILE"
```

Comment:
What is the reason that you setup a environment variable for this? Does the agent require this?

Reply:
yes, you are right! I don't think this is the optimal way it has been implemented. But as you said for consistency we can keep it as it is for now